pom.xml

```diff
@@ -43,18 +43,19 @@
   <properties>
     <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
     <gitRepositoryName>sonarlint-cli</gitRepositoryName>
+    <sonarlint.version>2.5.0.36</sonarlint.version>
```

Comment:
Don't forget to create a ticket for that. Also, we should in theory look at all the issues resolved in core that would benefit to SLCLI and create a dedicated ticket in SLCLI for each of them so that release notes of SLCLI are "complete".